aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferStatusUpdater.java

```diff
@@ -78,7 +78,7 @@
     /**
      * Prefix for temporary File created when client uploads an `InputStream`.
      */
```

Comment:
```suggestion
    /**
     * Prefix for temporary File created when client uploads an InputStream.   When the 
     * upload completes, the File is deleted, if it was created by the library, but not if it was
     * provided by the client.   To ensure we don't delete any File objects provided by the client,
     * a random UUID is included in the prefix.
     */
```

Reply:
Agree that it's kinda important to add javadoc to it 😄